Microsoft.Toolkit.Uwp.Services/Services/MicrosoftGraph/MicrosoftGraphExtensions.cs

```diff
@@ -41,16 +41,7 @@ public static void AddTo(this IUserMessagesCollectionPage source, ObservableColl
         /// <param name="dest">A collection of Microsoft Graph recipients</param>
         public static void CopyTo(this string[] source, List<Recipient> dest)
```

Comment:
Should these methods have an obsolete tag?

Reply:
nikola, this was an amazing spot, thank you so much. fixed.